newproject01/newproject01/q1.c

```diff
@@ -0,0 +1,67 @@
+#define _CRT_SECURE_NO_WARNINGS
+#include <stdio.h>
+#define N 8
+
+isNumberInOrder(int[], int, int); /*Declaration of a function*/
+
+int main1()
+{
+	int arr[N], number, i, flag = 1, counter = 0;
+
+	while (flag)
+	{
+		printf("Enter %d integer numbers, please: ", N);
+
+		for (i = 0; i < N && flag; i++)
+			if (scanf("%d", &arr[i]) != 1)
+				return 1;
+
+		if (arr[0] == 0)
+		{
+			for (i = 0; i < N; i++)
+			{
+				if (arr[i] == 0)
+					counter++;
+			}
+		}
+		if (counter == N)
+		{
+			printf("Thank you, we need to finish");
+			flag = 0;
+			continue;
+		}
+		printf("Enter integer number, please: ");
```

Comment:
Add blank line to separate the if block and the printf here, they are not related to each other.